packages/relay-runtime/store/RelayReader.js

```diff
@@ -1415,17 +1424,29 @@ function getResolverValue(
 
   let resolverResult = null;
   let resolverError = null;
+
+  const shouldPassContextToLiveResolver = field.kind === 'RelayLiveResolver';
```

Comment:
As stated above, I think we can unconditionally pass context to resolvers. Does not need to be just live resolvers.